nemo_skills/mcp/servers/exa_tool.py

```diff
@@ -60,19 +65,19 @@ async def exa_websearch(
 
     if response.status_code != 200:
         return {"error": response.json()["error"]}
-    else:
-        return {"result": response.json()["answer"]}
+
+    return {"result": response.json()["answer"]}
 
 
 def main():
     # Parse CLI arguments
     parser = argparse.ArgumentParser(description="MCP server for Exa web search tool")
-    parser.add_argument("--exa-api-key", dest="exa_api_key", type=str, required=False, help="Exa API key")
+    parser.add_argument("--api-key", type=str, required=False, help="Exa API key")
```

Comment:
same here, can probably be removed in favor of env variables, but double check

Reply:
I've updated this CLI arg to default to reading from the env.